_posts/2016-08-01-aquifer.md

```diff
@@ -199,9 +198,6 @@ We need to add two new extensions so it will read:
 
 {% highlight json %}
 "extensions": {
-  "aquifer-drush": {
```

Comment:
@ccjjmartin I removed this `aquifer-drush` setup to simplify things since you're not actually using this plugin in your tutorial.